code/datums/keybinding/_keybindings.dm

```diff
@@ -21,3 +21,15 @@
 /datum/keybinding/proc/intercept_mouse_special(datum/source)
 	SIGNAL_HANDLER
 	return COMSIG_MOB_CLICK_CANCELED
+
+/// returns the keys that the keybinding is currently triggered by
+/datum/keybinding/proc/get_keys_formatted(client/user)
+	if(!user)
+		return ""
+	var/datum/preferences/user_prefs = GLOB.preferences_datums[user.ckey]
+	if(!user_prefs)
+		return ""
+	for(var/key in user_prefs.key_bindings)
+		if(name in user_prefs.key_bindings[key])
+			. = "[key]"
```

Comment:
shouldnt this return immediately?

Reply:
forgot to change it back since it was working with key lists.